src/server/admin/server/convert1_9.go

```diff
@@ -203,19 +203,30 @@ func convert1_9Secrets(secrets []*pps1_9.Secret) []*pps.SecretMount {
 	return result
 }
 
+func copyStringSlice(ss []string) []string {
+	if ss == nil {
+		return nil
+	}
+	result := make([]string, 0, len(ss))
+	for _, s := range ss {
+		result = append(result, s)
+	}
+	return result
+}
+
 func convert1_9Transform(t *pps1_9.Transform) *pps.Transform {
 	if t == nil {
 		return nil
 	}
 	return &pps.Transform{
 		Image:            t.Image,
 		Cmd:              t.Cmd,
-		ErrCmd:           nil,
+		ErrCmd:           copyStringSlice(t.ErrCmd),
```

Comment:
Is there a reason we need to do a copy here but not on `t.Cmd` it seems like they should be being handled the same. My guess is you don't need to copy here but I might be wrong.

Reply:
Yeah, I guess you don't. I was thinking the conversion functions might need to do a deep copy, but I guess they are just there so particular conversions can be applied in one place.